mobile/src/main/res/values-pl/strings.xml

```diff
@@ -71,16 +70,7 @@
   <string name="error_certificate_expired">Certyfikat SSL jest przestarzały. Proszę uzyskaj nowy certyfikat lub wyłącz sprawdzanie poprawności certyfikatu podczas uzgadniania SSL</string>
   <string name="error_certificate_not_valid_yet">Certyfikat SSL nie jest jeszcze poprawny. Proszę uzyskaj poprawny certyfikat lub wyłącz sprawdzanie poprawności certyfikatu podczas uzgadniania SSL. Sprawdź również, czy twój telefon jest poprawnie ustawiony</string>
   <string name="error_certificate_revoked">Certyfikat SSL został unieważniony. Proszę uzyskaj nowy certyfikat lub wyłącz sprawdzanie poprawności certyfikatu podczas uzgadniania SSL</string>
-  <string name="error_http_code_401">Autoryzacja użytkownika nie powiodła się. Sprawdź nazwę użytkownika, hasło i certyfikat klienta SSL (HTTP response code 401)</string>
```

Comment:
@kubawolanin These strings contain english strings. That's why I un-approved them.